modulus/models/afno/afno.py

```diff
@@ -340,26 +340,26 @@ class PatchEmbed(nn.Module):
 
     Parameters
     ----------
-    img_size : Tuple[int, int]
-        Input image dimensions (height, width)
+    inp_shape : List[int]
+        Input image dimensions [height, width]
     in_channels : int
         Number of input channels
-    patch_size : Tuple[int, int], optional
-        Size of image patches, by default (16, 16)
+    patch_size : List[int], optional
+        Size of image patches, by default [16, 16]
     embed_dim : int, optional
         Embedded channel size, by default 256
     """
 
     def __init__(
         self,
-        img_size: Tuple[int, int],
+        inp_shape: List[int],
```

Comment:
IIUC, AFNO doesn't support more than two dimensional data so should add an `assert len(inp_shape) == 2`

Reply:
Added